src/main/java/org/pesmypetcare/webservice/entity/PetEntity.java

```diff
@@ -12,4 +17,57 @@
     private Double recommendedKcal;
     private String profileImageLocation;
     private String calendarId;
+
+    public PetEntity() { }
+
+    public PetEntity(GenderType gender, String breed, String birth, String pathologies, String needs,
+                     Double recommendedKcal, String profileImageLocation, String calendarId) {
+        checkDateFormat(birth);
+        this.gender = gender;
+        this.breed = breed;
+        this.birth = birth;
+        this.pathologies = pathologies;
+        this.needs = needs;
+        this.recommendedKcal = recommendedKcal;
+        this.profileImageLocation = profileImageLocation;
+        this.calendarId = calendarId;
+    }
+
+    /**
+     * Checks that field, key and body have the correct format of a Pet attribute.
+     * @param field Name of the attribute collection. Possible fields: meals, trainings, washes, weights
+     * @param key Key of the attribute
+     * @param body Body of the attribute
+     */
+    public static void checkKeyAndBody(String field, String key, Map<String, Object> body) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [NCSS for this method is 16 (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5469160)